src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -994,7 +994,7 @@ void Instruction::execute() {
         break;
       }
       case Opcode::AArch64_BRK: {
-        return executionNYI();
+        // TODO: Generate breakpoint exception
```

Comment:
I fear that having this as a TODO comment rather than an NYI exception could cause issues in the future.

Reply:
I have added a new exception type and thrown it here, but it remains unhandled. I would be unsure about the best way to handle the exception so perhaps we can chat about it?